ckanext/geodatagov/templates/source/geodatagov_source_form.html

```diff
@@ -50,7 +50,7 @@
 
 </fieldset>
 
-{% set private_datasets = source_config.get('private_datasets') or data.private_datasets %}
+{% set private_datasets = source_config.get('private_datasets') or data.private %}
```

Comment:
Not sure about this. 
If a harvest source is private: should we consider private all datasets harvested from them?

Reply:
So, this would change ALL datasets from all sources from this organization or all datasets from this harvest source?

If it's the latter then I think this makes sense, right?

Reply:
This form is about a harvest source. `data` here is a DataGovHarvest. I think organization are not related to this form.
The config value `private_datasets` is the one (and I think is the only one) that defines if harvested dataset from it should be public or not. 
The `private` property is inherited from packages. The harvest sources are also CKAN packages. This property is about to show or hides the harvest source itself. 
If `data.private` should immediately change the visibility of all datasets from this source, we will need to work on this in a different way.

Reply:
Hmm, this seems confusing and arbitrary to me to have both. Would there be any harm in just removing `data` and only using the `source_config`?

Reply:
For me also seems arbitrry. `private_datasets` [was added as an `extra-schema` field](https://github.com/GSA/ckanext-geodatagov/commit/2406ff2c3317378cb81e1c1ed448260ae7752bae) and we need to analyze deeply how `extra_schemas` works before removing this.

Reply:
Flipping the order of our `or` might be the correct solution if we can't remove `data.private_datasets`

Reply:
Unless we want private_datasets to be set to `None`? Then we need to change the `if` in the form to:

{% if private_datasets != None %}selected='Public'{% endif %}

Reply:
I think we close this PR without merging. I'm not sure why, but the `selected="selected"` doesn't populate in catalog classic but does in catalog-next. See the screen shots at the end of [this issue](https://github.com/GSA/datagov-deploy/issues/897).

I don't think it makes sense to make my proposed changes since this is fixed in  the new version.

Reply:
Ok, I'll close this without merging. Maybe you can create a PR just for CKAN 2.3 version